plugins/remote/enroll/tls_enroll.cpp

```diff
@@ -35,9 +35,15 @@ DECLARE_bool(disable_enrollment);
 
 CLI_FLAG(uint64,
          tls_enroll_max_attempts,
-         3,
-         "Number of attempts to retry a TLS enroll request, it used to be the "
-         "same as [config_tls_max_attempts]");
+         12,
+         "The total number of attempts that will be made to the remote enroll "
```

Comment:
Do you mind making this description more succinct? The maximum character count (on Linux at least) is 116. This is 184. I think we should expect people to refer to the documentation page for the more descriptive/complete version.

My recommendation is: `The total number of attempts that will be made to the enroll endpoint if a request fails, 0 for infinite`

Reply:
@theopolis done!